tools/Microsoft.TemplateSearch.TemplateDiscovery/Test/CacheFileTests.cs

```diff
@@ -84,7 +84,7 @@ internal static void RunTests(string metadataPath, string legacyMetadataPath)
             CanSearch(workingDirectory, metadataPath);
 
             //7.0.300
-            sdkVersion = "7.0.300";
+            sdkVersion = "7.0.300-preview.23068.14";
```

Comment:
this won't work - on next build the version will change.

Reply:
the test is disabled